tools/validators/instance_validator/validate/telemetry_error.py

```diff
@@ -39,8 +39,8 @@ def __eq__(self, other):
   def GetPrintableMessage(self):
     """Returns a human-readable message that explains this error."""
 
-    msg = "Error for entity [" + self.entity + "]"
-    if self.point:
-      msg += ", point [" + self.point + "]"
-    msg += ": " + self.message
+    msg = "- entity [" + self.entity + "]"
```

Comment:
I'm in favor of removing the "Error for" prefix to decrease redundancy in the error report. However, can we also ditch the "- " prefix and rely on the report generation code to format properly? This message might be used in other contexts where we don't want to have any prefix.